libnavigation-core/api/current.txt

```diff
@@ -0,0 +1,443 @@
+  }
+
+  public static final class FasterRouteObserver.Companion {
+    method public long getDEFAULT_INTERVAL_MILLIS();
+    property public final long DEFAULT_INTERVAL_MILLIS;
+  }
+
+}
+
+package com.mapbox.navigation.core.replay {
+
+  public final class MapboxReplayer {
+    ctor public MapboxReplayer();
+    method public void clearEvents();
+    method public double durationSeconds();
+    method public void finish();
+    method public void play();
+    method public void playFirstLocation();
+    method public void playbackSpeed(double scale);
+    method public com.mapbox.navigation.core.replay.MapboxReplayer pushEvents(java.util.List<? extends com.mapbox.navigation.core.replay.history.ReplayEventBase> events);
+    method public void pushRealLocation(android.content.Context context, double eventTimestamp);
+    method public void registerObserver(com.mapbox.navigation.core.replay.history.ReplayEventsObserver observer);
+    method public void seekTo(double replayTime);
+    method public void seekTo(com.mapbox.navigation.core.replay.history.ReplayEventBase replayEvent);
+    method public void stop();
+    method public boolean unregisterObserver(com.mapbox.navigation.core.replay.history.ReplayEventsObserver observer);
+    method public void unregisterObservers();
+  }
+
+  public final class ReplayLocationEngine implements com.mapbox.android.core.location.LocationEngine com.mapbox.navigation.core.replay.history.ReplayEventsObserver {
+    ctor public ReplayLocationEngine(com.mapbox.navigation.core.replay.MapboxReplayer mapboxReplayer);
+    method public void getLastLocation(com.mapbox.android.core.location.LocationEngineCallback<com.mapbox.android.core.location.LocationEngineResult> callback);
+    method public void removeLocationUpdates(com.mapbox.android.core.location.LocationEngineCallback<com.mapbox.android.core.location.LocationEngineResult> callback);
+    method public void removeLocationUpdates(android.app.PendingIntent? pendingIntent);
+    method public void replayEvents(java.util.List<? extends com.mapbox.navigation.core.replay.history.ReplayEventBase> replayEvents);
+    method public void requestLocationUpdates(com.mapbox.android.core.location.LocationEngineRequest request, com.mapbox.android.core.location.LocationEngineCallback<com.mapbox.android.core.location.LocationEngineResult> callback, android.os.Looper? looper);
+    method public void requestLocationUpdates(com.mapbox.android.core.location.LocationEngineRequest request, android.app.PendingIntent? pendingIntent);
+    field public static final com.mapbox.navigation.core.replay.ReplayLocationEngine.Companion! Companion;
+  }
+
+  public static final class ReplayLocationEngine.Companion {
+    method public int getInstances();
+    method public void setInstances(int p);
+    property public final int instances;
+  }
+
+  public final class ReplayLocationEngineKt {
```

Comment:
Could we avoid tracking this empty `public` class? 🤔

Reply:
Yeah removing that https://github.com/mapbox/mapbox-navigation-android/pull/3142